### PR TITLE
sl-62: start page

### PR DESCRIPTION
**Pull request by Alyona-Pisotska**
image

### CODE REVIEW

File: frontend/src/pages/start/start.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Logo, PageLayout, Stepper } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+const StartPage: React.FC = () => {
+  return (
+    <PageLayout style="black" className={styles['layout']}>
+      <div className={styles['page-wrapper']}>
+        <Logo />
```
Comment (what1s1ove) [inline frontend/src/pages/start/start.tsx]: we should have a full header here

File: frontend/src/libs/components/logo/logo.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.scss';
+
+const Logo: React.FC = () => {
```
Comment (what1s1ove) [inline frontend/src/libs/components/logo/logo.tsx]: left it in the header

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/pages/start/start.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Logo, PageLayout, Stepper } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+const StartPage: React.FC = () => {
```
Comment (what1s1ove) [inline frontend/src/pages/start/start.tsx]:
```suggestion
const Start: React.FC = () => {
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/libs/enums/app-route.enum.ts [hunk 4]
```diff
@@ -2,6 +2,7 @@ const AppRoute = {
   ROOT: '/',
   SIGN_IN: '/sign-in',
   SIGN_UP: '/sign-up',
+  START: 'start',
```
Comment (andanf-e) [inline frontend/src/libs/enums/app-route.enum.ts]: Missed slash?

**Review by andanf-e — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/logo/logo.tsx (hunk 2 above)
Comment (andanf-e) [reply frontend/src/libs/components/logo/logo.tsx]: Looks like it was not resolved

**Review by andanf-e — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/logo/logo.tsx (hunk 2 above)
Comment (Alyona-Pisotska) [reply frontend/src/libs/components/logo/logo.tsx]: Don't need to create Logo component? I thought it was more correct

**Review by andanf-e — APPROVED** [review]
🔥

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**